### ISSUE TITLE
Indoor directions - accessibility feature not working

### ISSUE THREAD

**Issue by jainammshah12** [mcndl13/SOEN-390-Mini-Capstone#102]
### **Description**

Since the indoor directions aren't completely built, I was unable to apply accessibility features. So anyone who takes care of indoor directions would need to focus on integrating accessibility features as well.

### **Priority**

Low/Medium

**Comment by MCSTEVE1000** [comment]
Ok will keep that in mind when I'll be done with the indoor directions

**Comment by nicpuertas** [comment]
Completed
